lib/App/DuckDuckGo/UI.pm

```diff
@@ -530,7 +531,7 @@ sub default_bindings {
             my $this = shift;
             # Focus either the ZCI box or the Deep box if this is the last history entry already.
             $zci_box->hidden ? $deep_box->focus : $zci_box->focus, return if $this->history_index >= -1;
-            $this->history_index++;
```

Comment:
TBH, I'm surprised that even worked at all to start with. I wasn't aware Moo used lvalue subs.